src/schema/rules/sidecars/meg.yaml

```diff
@@ -35,12 +35,12 @@ MEGRecommended:
         For MEG scanners, this must be one of:
         `"CTF"`, `"Elekta/Neuromag"`, `"BTi/4D"`, `"KIT/Yokogawa"`,
         `"ITAB"`, `"KRISS"`, `"Other"`.
-        See [Appendix VII](SPEC_ROOT/99-appendices/07-meg-systems.md) for
+        See [Appendix VII](SPEC_ROOT/appendices/meg-systems.md) for
         preferred names.
     ManufacturersModelName:
       level: recommended
       description_addendum: |
-        See [Appendix VII](SPEC_ROOT/99-appendices/07-meg-systems.md) for
+        See [Appendix VII](SPEC_ROOT/appendices/meg-systems.md) for
```

Comment:
Actually, it looks like we're using this `Appendix <NUMERAL>` form a lot. If we're reordering, we also need to renumber.

Reply:
Thanks for catching these. I was planning to remove the numbers completely and just reference the appendices by name (e.g., "MEG Systems Appendix"). That way, we can reorder whenever we want without worrying about inaccurate numbering. WDYT?

Reply:
Fine with me. Possible @sappelhoff remembers a reason we keep it this way?

Reply:
I don't think there ever was a reason beyond "that's how appendices are often labeled in documents", +1 to refer to them by name instead. Nobody remembers "appendix V", "units appendix" is much more clear.

Reply:
Awesome! I've switched to the names.